db/db_impl/db_impl.cc

```diff
@@ -4710,8 +4710,19 @@ Status DBImpl::CreateColumnFamilyWithImport(
 
   import_job.Cleanup(status);
   if (!status.ok()) {
-    DropColumnFamily(*handle);
-    DestroyColumnFamilyHandle(*handle);
+    Status temp_s = DropColumnFamily(*handle);
+    if (!temp_s.ok()) {
+      ROCKS_LOG_ERROR(immutable_db_options_.info_log,
+                      "DropColumnFamily failed with error %s",
+                      temp_s.ToString().c_str());
+    }
+    if (versions_->io_status().IsIOError()) {
```

Comment:
Sorry for the confusion. Checked the code again. If `!io_status.ok()`, then `temp_s` must be non-ok, thus we just need to log once.